app/services/content/items_service.rb

```diff
@@ -15,6 +15,16 @@ def fetch_all_with_default_locale_only
       end
   end
 
+  def fetch_all_with_id_and_path
```

Comment:
The name of this method obfuscates behaviour that it implements. The `with_id_and_path` suffix suggests it will return a collection with `id` and `path` attributes where in fact it returns `content_id`, `base_path`, `title`,  and `description`. We only need `content_id` and `base_path`.

The previous method in the file has a `with_default_locale_only` suffix which reveals that we'll return the English version of the content item, or if that doesn't exist the first translation. We're not revealing that here. More importantly though, we don't want that behaviour, we only want to return content items that have an English locale.